src/docs/README.md

```diff
@@ -119,13 +135,16 @@ To add a sidebar element to the page, use the codeblock syntax:
   - Only headers up to `###` are parsed. All header with more depth are parsed to `###`.
 - Tries to match active browser url to referenced ``href`` of any element to set active page.
 
-## _ignored
 
-Anything inside this folder will not be used to generate html pages.
+## Public or not public
```

Comment:
Please rephrase these changes, not descriptive enough. 

For example i do not understand what the following is meant to confer.
```markdown
### visible or hidden

- sidebar
```

Reply:
Change: 
- `Public or not public` to `Markdown to HTML Rendering Rules`
- Captitalize non-filler words in headers for consistency:
  - Ignored **C**ontent
  - **V**isible or **H**idden